src/client/errors.ts

```diff
@@ -1,30 +1,24 @@
 import { ErrorPolykey, sysexits } from '../errors';
 
-class ErrorClient<T> extends ErrorPolykey<T> {}
+class ErrorRPCClient<T> extends ErrorPolykey<T> {}
 
-class ErrorClientClientDestroyed<T> extends ErrorClient<T> {
-  static description = 'GRPCClientClient has been destroyed';
-  exitCode = sysexits.USAGE;
-}
-
-class ErrorClientAuthMissing<T> extends ErrorClient<T> {
+class ErrorClientAuthMissing<T> extends ErrorRPCClient<T> {
   static description = 'Authorisation metadata is required but missing';
   exitCode = sysexits.NOPERM;
 }
 
-class ErrorClientAuthFormat<T> extends ErrorClient<T> {
+class ErrorClientAuthFormat<T> extends ErrorRPCClient<T> {
   static description = 'Authorisation metadata has invalid format';
   exitCode = sysexits.USAGE;
 }
 
-class ErrorClientAuthDenied<T> extends ErrorClient<T> {
+class ErrorClientAuthDenied<T> extends ErrorRPCClient<T> {
   static description = 'Authorisation metadata is incorrect or expired';
   exitCode = sysexits.NOPERM;
 }
 
 export {
-  ErrorClient,
-  ErrorClientClientDestroyed,
+  ErrorRPCClient,
   ErrorClientAuthMissing,
   ErrorClientAuthFormat,
   ErrorClientAuthDenied,
```

Comment:
All these should the be `ErrorRPCClientX`. Always keep to consistent naming.

On the other hand, originally I just called it `ErrorClient` to avoid the `RPC`. You can also change `ErrorRPCClient` to just `ErrorClient`.

Reply:
This meaning it is an error inside the client service.

Reply:
I don't think so? They are different domains. These are the Client errors for the `client` domain., the other ones are `rpc` domain errors for the RPCClient. They are completely different.

Reply:
Yea so then `ErrorRPCClient` should be `ErrorClient`?

Reply:
You're extending `ErrorRPCClient` atm, and the subtypes are just `ErrorClientX`. So instead it should be `ErrorClientX` extending `ErrorClient`.

Reply:
Ah, I didn't see that in the context of the comment. I'll fix it.